airbyte-integrations/connectors/source-gainsight-px/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+data:
+  allowedHosts:
+    hosts:
+      - TODO # Please change to the hostname of the source.
+  registries:
+    oss:
+      enabled: false
+  connectorSubtype: api
+  connectorType: source
+  definitionId: 0da3b186-8879-4e94-8738-55b48762f1e8
+  dockerImageTag: 0.1.0
+  dockerRepository: airbyte/source-gainsight-px
+  githubIssueLabel: source-gainsight-px
+  icon: gainsight-px.svg
```

Comment:
Please add the icon

Reply:
Could you please provide the path for adding the icon, as the config-oss directory has recently changed

Reply:
Added in the same directory :)